helm/openidm/templates/tls.yaml

```diff
@@ -1,20 +1,22 @@
-{{- if .Values.useTLS }}
+# CertManager certificate object created if 'tls.strategy: CertManager'. 
+# This object triggers a certificate request to LetsEncrypt via the CertManager Operator.
```

Comment:
Is it "LetsEncrypt" or "letsencrypt"? Have a look through your comments and pls make this consistent one way or the other.